app/javascript/controllers/ocr_controller.js

```diff
@@ -122,59 +187,4 @@ export default class extends Controller {
       `;
     }).join('');
   }
-
-  changed () {
-    if (this.contentStructureTarget && this.ocrAllowed().indexOf(this.contentStructureTarget.value) > -1) {
-      this.ocrSettingsTarget.hidden = false;
-      this.runOcrTarget.querySelector('legend').innerHTML = this.labelRunOcr();
-      this.manuallyCorrectedOcrTarget.querySelector('legend').innerHTML = this.labelManuallyCorrected();
-    } else {
-      this.ocrSettingsTarget.hidden = true;
-    }
-  }
-
-  // if the user indicates they want to runn SDR OCR, show any relevant notes/warnings
-  run_ocr_changed() {
-    const runocr = this.runOcrTarget.querySelector('input[type="radio"]:checked').value == 'true';
-    if (runocr)
-    {
-      const isdocument = this.selected_content_structure().value == 'document';
-      this.runOcrDocumentNotesTarget.hidden = !isdocument;
-      this.ocrLanguageWrapperTarget.classList.remove('d-none');
-    }
-    else
-    {
-      this.runOcrDocumentNotesTarget.hidden = true;
-      this.ocrLanguageWrapperTarget.classList.add('d-none');
-    }
-    Object.values(this.ocrDropdownTarget.children).forEach(child => {
-      child.disabled = !runocr;
-    })
-  }
-
-  content_structure_changed () {
-    // Hide the OCR settings if the selected content structure is not in the list of allowed content structures
-    if (this.ocrAllowed().indexOf(this.contentStructureTarget.value) < 0) {
-      this.ocrSettingsTarget.hidden = true
-      return
-    }
-
-    // Show the OCR settings if the selected content structure is in the list of allowed content structures
-    this.ocrSettingsTarget.hidden = false
-
-    // Set the OCR settings label
-    this.runOcrTarget.querySelector('legend').innerHTML = this.labelRunOcr()
-
-    // Set specific OCR labels/options based on the content structure
-
-    if (this.selected_content_structure().value == 'document') {
-      this.manuallyCorrectedOcrTarget.querySelector('legend').innerHTML = this.labelDocumentsManuallyCorrected()
-      this.manuallyCorrectedOcrOptionsTargets[1].labels[0].innerHTML = "No/Don't Know" // the "No" option label for manually corrected document OCR
-      this.run_ocr_changed()
-    } else {
-      this.manuallyCorrectedOcrTarget.querySelector('legend').innerHTML = this.labelImagesManuallyCorrected()
-      this.manuallyCorrectedOcrOptionsTargets[1].labels[0].innerHTML = 'No' // the "No" option label for manually corrected image OCR
-      this.runOcrDocumentNotesTarget.hidden = true
-    }
-  }
```

Comment:
all of this stuff was adjusted and moved higher in the file so that the OCR selection JS is together and the language related JS is together